pom.xml

```diff
@@ -44,10 +44,10 @@ THE SOFTWARE.
   </licenses>
     
   <properties>
-    <jenkins.version>1.609.3</jenkins.version>
+    <jenkins.version>1.642.3</jenkins.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
Yup yup - at one point, I was experimenting with stuff that did need newer versions of other dependencies and bumped things, but I'll roll this back now.